source/guides/guides/test-retries.md

```diff
@@ -207,3 +207,24 @@ No. Tests recorded during `cypress run` with the `--record` flag will be counted
 We consider each time the `it()` function is called to be a single test for billing purposes. The test retrying will not count as extra test recordings in your billing.
 
 You can always see how many tests you've recorded from your organization's Billing & Usage page within the {% url "Dashboard" https://on.cypress.io/dashboard %}.
+
+## Can I access the current attempt counter from the test?
+
+Yes, although ordinarily you would not have to, since this is a low-level detail. But if you want to use the current attempt number and the total allowed attempts you could do the following:
+
+```javascript
+it('passes on the last attempt', { retries: 3 }, () => {
+  // cy.state('runnable') returns the current test object
+  // we can grab the current attempt and
+  // the total allowed attempts from its properties
+  const attempt = cy.state('runnable')._currentRetry
+  const retries = cy.state('runnable')._retries
+
+  if (attempt < retries) {
+    expect(true).to.be.false // fail the test
+  }
+  // passes on the last attempt
```

Comment:
I think including this assertion is just going to mislead people when they copy/paste this - as this is not a use case anyone should ever need to actually write. 

```suggestion
```